### PR TITLE
added metadata for con4gis/ldap and con4gis/reservation

### PR DESCRIPTION
**Pull request by coastforge-mei**

### CODE REVIEW

File: meta/con4gis/reservation/de.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+de:
+    title: con4gis-Reservation
+    description: >
+        Der Baustein ist für Tisch- und Raumreservierungen bereits im Einsatz. Beliebiege reservierbare Objekte können im Contao Backend definiert werden.
```
Comment (xchs) [inline meta/con4gis/reservation/de.yml:4]:
```suggestion
        Der Baustein ist für Tisch- und Raumreservierungen bereits im Einsatz. Beliebige reservierbare Objekte können im Contao Backend definiert werden.
```

File: meta/con4gis/reservation/de.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        Aufgrund der Corona Krise und dem akuten Bedarf (z.B. in der Gastronomie) haben wir uns dazu entschlossen auch diese Komponente zu veröffentlichen.
```
Comment (xchs) [inline meta/con4gis/reservation/de.yml:5]:
```suggestion
        Aufgrund der Corona Krise und dem akuten Bedarf (z. B. in der Gastronomie) haben wir uns dazu entschlossen, auch diese Komponente zu veröffentlichen.
```

File: meta/con4gis/reservation/en.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+en:
+    title: con4gis Reservation
+    description: >
+        The module is already in use for table and room reservations. Any objects that can be reserved can be defined in the Contao backend.
```
Comment (xchs) [inline meta/con4gis/reservation/en.yml:4]:
```suggestion
        The module is already in use for table and room reservations. Any objects that can be reserved can be defined in the Contao back end.
```

File: linter/whitelists/de.txt [hunk 4]
```diff
@@ -30,6 +30,7 @@ ausserdem
 Automatisierungsaufgaben
 Avatar
 Avatare
+Backend
```
Comment (xchs) [inline linter/whitelists/de.txt:33]: Was that complained by the Linter?

Comment (coastforge-mei) [reply linter/whitelists/de.txt:33]: Just got ldap and reservation into it. The "Backend" entry seems to have been in our fork for a while. Can't tell if the linter was complaining.

**Comment by xchs** [comment]
Please update/rebase your master branch. I cannot merge 67 commits, most of which are unrelated to this PR.

**Comment by coastforge-mei** [comment]
We have set up a new fork and pull request. The rebase causes problems every time. I hope that the new fork will fix this.

**Pull request closed without merging**